### PR TITLE
Enables A Substantial Number of Disabled Overmaps Files, Updates Adjacent Files and Dependencies, and Attempts to Establish Feature Parity.

### PR DESCRIPTION
**Pull request by Captain277**
## About The Pull Request

1. _Ticks a large number of unticked files and establishes or repairs their dependencies. This is being done to hopefully assist Kevin's overmap improvement work._

## Why It's Good For The Game

1. Front-loading and editing for larger planned features. Honestly I barely know what I'm doing here and I'm just kinda hoping this all shakes out.

## Changelog
:cl:
add: Adds or ticks multiple overmaps related files.
tweak: Edits multiple files to match dependencies.
/:cl:

### CODE REVIEW

File: code/_globalvars/misc.dm [hunk 1]
```diff
@@ -16,3 +16,9 @@ GLOBAL_VAR(bible_icon_state)
 GLOBAL_VAR(bible_item_state)
 GLOBAL_VAR(holy_weapon_type)
 GLOBAL_VAR(holy_armor_type)
+
+//Vore Port
+GLOBAL_LIST_EMPTY(error_last_seen)
+GLOBAL_LIST_EMPTY(error_cooldown)
+
+GLOBAL_DATUM_INIT(destroyed_event, /decl/observ/destroyed, new())
```
Comment (silicons) [inline code/_globalvars/misc.dm:24]: needs to go

File: code/datums/observation/destroyed.dm [hunk 2]
```diff
@@ -0,0 +1,15 @@
+//	Observer Pattern Implementation: Destroyed
```
Comment (silicons) [inline code/datums/observation/destroyed.dm]: entire file needs to go

File: code/__DEFINES/dcs/signals.dm [hunk 3]
```diff
@@ -127,8 +127,10 @@
 // /turf signals
 #define COMSIG_TURF_CHANGE "turf_change"						//from base of turf/ChangeTurf(): (path, list/new_baseturfs, flags, list/transferring_comps)
 #define COMSIG_TURF_HAS_GRAVITY "turf_has_gravity"				//from base of atom/has_gravity(): (atom/asker, list/forced_gravities)
-#define COMSIG_TURF_MULTIZ_NEW "turf_multiz_new"				//from base of turf/New(): (turf/source, direction)
 */
+#define COMSIG_TURF_MULTIZ_DEL "turf_multiz_del"
+///from base of turf/multiz_turf_new: (turf/source, direction)
+#define COMSIG_TURF_MULTIZ_NEW "turf_multiz_new"
```
Comment (silicons) [inline code/__DEFINES/dcs/signals.dm:133]: these need to be implemented 
tl;dr
copypasting the defines in don't mean anything if they're not fired using SEND_SIGNAL 
find where it's doing this in virgo and send hte signal, or anything using this won't ever receive them

File: code/_globalvars/misc.dm (hunk 1 above)
Comment (Captain277) [reply code/_globalvars/misc.dm:24]: Lines removed.

File: code/datums/observation/destroyed.dm (hunk 2 above)
Comment (Captain277) [reply code/datums/observation/destroyed.dm]: File deleted.

File: code/__DEFINES/dcs/signals.dm (hunk 3 above)
Comment (Captain277) [reply code/__DEFINES/dcs/signals.dm:133]: Signals defined.
